sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala

```diff
@@ -684,17 +684,17 @@ class JDBCSuite extends SparkFunSuite
   }
 
   test("quote column names by jdbc dialect") {
```

Comment:
I understand the indention is consistency, but let's keep the unrelated code untouched in this PR.
We can do refactoring PR later if needed.